### PR TITLE
Dev set donmain name

### PR DESCRIPTION
**Pull request by raywang1223**

### CODE REVIEW

File: cloud/build.gradle [hunk 1]
```diff
@@ -84,11 +84,12 @@ dependencies {
     compile("org.apache.httpcomponents:httpmime:$httpclient_version") {
         exclude(group: 'org.apache.httpcomponents', module: 'httpclient')
     }
-    compile project(':swagger-android-client')
+    // compile project(':swagger-android-client')
```
Comment (bipoegg) [inline cloud/build.gradle]: remove unused code.

File: cloud/src/main/java/uk/co/siterwell/cloud/Endpoint.java [hunk 2]
```diff
@@ -0,0 +1,60 @@
+package uk.co.siterwell.cloud;
+
+/**
+ * Created by raywang on 16/4/13.
+ */
+public enum Endpoint {
+
+    QA,PRO,DEV;
```
Comment (bipoegg) [inline cloud/src/main/java/uk/co/siterwell/cloud/Endpoint.java]: What this mean?

File: cloud/build.gradle (hunk 1 above)
Comment (raywang1223) [inline cloud/build.gradle]: done

File: cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java [hunk 3]
```diff
@@ -154,6 +161,15 @@ private void initializeProperty() {
...
+
+        // init cloud settings
+        settings = context.getSharedPreferences(CONFIG, 0);
+
+        // init url settings
+        baseUrl = Endpoint.toBaseUrl(getEndpoint());
+        mqUrl = Endpoint.toMqUrl(getEndpoint());
+
+        setUrlToHelper();
     }
```
Comment (bipoegg) [inline cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java]: When initializeProperty is called? onResume or onCreate?

File: cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java [hunk 4]
```diff
@@ -154,6 +161,15 @@ private void initializeProperty() {
         deviceHelper = new DeviceHelper();
         ruleHelper = new RuleHelper();
         utilityHelper = new UtilityHelper();
+
+        // init cloud settings
+        settings = context.getSharedPreferences(CONFIG, 0);
+
+        // init url settings
+        baseUrl = Endpoint.toBaseUrl(getEndpoint());
+        mqUrl = Endpoint.toMqUrl(getEndpoint());
```
Comment (bipoegg) [inline cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java]: if update from earlier version, it have no Setting, do we need another default value?

File: cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java (hunk 3 above)
Comment (raywang1223) [inline cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java]: onCreate

Comment (raywang1223) [inline cloud/src/main/java/uk/co/siterwell/cloud/CloudHandler.java]: when instance is init

**Pull request merged**